core/src/main/java/com/sequenceiq/cloudbreak/service/ServiceEndpointCollector.java

```diff
@@ -255,7 +267,7 @@ private String getHiveJdbcUrlFromGatewayTopology(String ambariIp, GatewayTopolog
     private String getHdfsUIUrlWithHostParameterFromGatewayTopology(String ambariIp, GatewayTopology gt, String nameNodePrivateIp) {
         Gateway gateway = gt.getGateway();
         String url = String.format("https://%s:%s/%s/%s%s?host=http://%s:%s", ambariIp, knoxPort, gateway.getPath(), gt.getTopologyName(),
-                ExposedService.NAMENODE.getKnoxUrl(), nameNodePrivateIp, ExposedService.NAMENODE.getAmbariPort());
+                ExposedService.NAMENODE.getKnoxUrl(), nameNodePrivateIp, ExposedService.NAMENODE.getCmPort());
```

Comment:
Like above, I have a feeling this will not work with Ambari anymore.

Reply:
Answer is the same as above:)

Reply:
Acknowledged. 😃